manifest.toml

```diff
@@ -19,7 +19,8 @@ code = "https://github.com/nextcloud/server"
 cpe = "cpe:2.3:a:nextcloud:nextcloud"
 
 [integration]
-yunohost = ">= 11.2.30"
+yunohost = ">= 12.0.7"
```

Comment:
Might need ugh 12.0.8 ? (for the mysql helper stuff which is ... not released yet)